deserialize java.util.Arraylist cause NPE

We are using Kryo 2.24 for message serialization and deserialization, but it will throws a NPE when encounter java.util.Arraylist
```Caused by: java.lang.NullPointerException
    at java.util.ArrayList.ensureExplicitCapacity(ArrayList.java:215)
    at java.util.ArrayList.ensureCapacity(ArrayList.java:199)
    at com.esotericsoftware.kryo.serializers.CollectionSerializer.read(CollectionSerializer.java:96)
    at com.esotericsoftware.kryo.serializers.CollectionSerializer.read(CollectionSerializer.java:22)
    at com.esotericsoftware.kryo.Kryo.readObject(Kryo.java:679)
    at com.esotericsoftware.kryo.serializers.ObjectField.read(ObjectField.java:106) ```

I also found some related issue in Kryo: 
https://github.com/EsotericSoftware/kryo/issues/254

Kryo 3.0 is released and the problem seems is resolved, I'm not sure should we upgrade the Kryo version?

--- a/core/src/main/scala/org/apache/gearpump/serializer/FastKryoSerializer.scala
+++ b/core/src/main/scala/org/apache/gearpump/serializer/FastKryoSerializer.scala
@@ -19,6 +19,7 @@
 package org.apache.gearpump.serializer
 
 import akka.actor.{ExtendedActorSystem}
+import com.esotericsoftware.kryo.Kryo.DefaultInstantiatorStrategy
 import com.romix.akka.serialization.kryo.{KryoSerializer}
 import org.apache.gearpump.serializer.FastKryoSerializer.KryoSerializationException
 import org.apache.gearpump.util.LogUtil
@@ -30,6 +31,7 @@ class FastKryoSerializer(system: ExtendedActorSystem) {
 
   private val kryoSerializer = new KryoSerializer(system).serializer
   private val kryo = kryoSerializer.kryo
+  kryo.setInstantiatorStrategy(new DefaultInstantiatorStrategy)
   private val kryoClazz = new GearpumpSerialization(config).customize(kryo)
 
 

